Adding README file.

mithro opened the pull request:
README is based on @suzyh's notes on how to get running.

mithro committed:
Adding README.md file

Instructions from @suzyh.

mithro committed:
Adding missing LICENSE file.

mithro committed:
More fixes to the README.

dstoc commented on README.md:
````diff
@@ -0,0 +1,53 @@
+
+Install [Polymer](https://www.polymer-project.org/1.0/docs/start/getting-the-code.html)
+```shell
+bower install --save Polymer/polymer
````
I think you just want to install the bower dependencies with `bower install`

`bower install --save Polymer/polymer` will install the latest version of polymer and then update the `bower.json` file to reference that (but any other dependencies would not be installed).

mithro replied:
Done.

mithro committed:
Fix the install instructions.

shans commented on bower.json:
```diff
@@ -16,7 +16,7 @@
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
+    "polymer": "Polymer/polymer#^1.4.0",
```
Don't change this dependency as part of this pull request.

mithro replied:
Done.

shans commented on README.md:
```diff
@@ -0,0 +1,53 @@
+
+ * Consoles
+   - ["chromez-app" project](http://console.developers.google.com/)
+   - ["dashboard"](https://pantheon.corp.google.com/home/dashboard?project=chromez-app)
```
I don't think we want this link here...

suzyh replied:
Uh, yeah. These instructions started out as my notes to myself. I didn't expect them to end up in a public README.

mithro replied:
I changed the link to the external equivalent and rewrote history to remove it.

shans commented:
I think to make this mergable we'd need to:
- Add a new section at the top that points to chromez-app.appspot.com and talks about how you can load configs from arbitrary URLs.
- Add a section describing the config format.
- Move all of the 'running, etc' stuff into a sub-section further down the page under a heading 'if you want to hack on chromez' (or similar)
- Remove the 'deploying' section entirely.

I understand this is a fair amount of work. If you decide to just close this PR that's fine - we can sort something out later.

alancutter commented on README.md:
```diff
@@ -0,0 +1,53 @@
+
+## Deploying
+
+If you have access to the chromez app itself:
```
This needs to include downloading the third party libraries: `pip install -t lib google-api-python-client`

mithro replied:
Done.

mithro committed:
Adding FIXME for @shans.

mithro committed:
Adding library to deploy instructions.

mithro commented:
Added a section at top for you to fill in with the details when you get around to it.
Move the stuff under a "Developer Instructions".
Fixed the deploying instructions by linking to the external url.

shans commented:
I don't want us to have deploy instructions up at all. Only a tiny handful of people will ever be allowed to deploy to chromez-app.

mithro commented:
Ask discussed at your desk, I have made the deploy instructions generic for deploying to your own App Engine instance.

shans commented:
Now they're just generic deploy instructions for how to deploy any app-engine app. There's already documentation for this. Just remove the deploy section please.

alancutter commented:
It's nice to have the google-api-python-client step covered, we got caught out on that one earlier as we didn't know about the dependency.

It'd be good to mention running `bower install` before uploading as well in case the bower dependencies have changed (again speaking from experience).

shans commented:
Again, I'm perfectly happy to have specific instructions for chromez-app in a private doc that is shared among the maintainers (i.e. Doug, Suzy, I). They don't need to be here.

mithro committed:
Remove deploy instructions.

mithro commented:
I've removed the deploy instructions as requested.

shans commented:
Thanks!
